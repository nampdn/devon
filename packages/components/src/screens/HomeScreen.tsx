import React from 'react';
import {Text, View} from 'react-native';

export const HomeScreen = React.memo(() => {
  return (
    <View>
      <Text>Hello Todo App</Text>
    </View>
  );
});
