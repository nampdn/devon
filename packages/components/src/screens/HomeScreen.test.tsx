import React from 'react';
import {create} from 'react-test-renderer';
import {HomeScreen} from './HomeScreen';

describe('HomeScreen', () => {
  it('render and match snapshot', () => {
    const tree = create(<HomeScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
