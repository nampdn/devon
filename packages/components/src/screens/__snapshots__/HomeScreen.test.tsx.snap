// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeScreen render and match snapshot 1`] = `
<View>
  <Text>
    Hello Todo App
  </Text>
</View>
`;
